cmd/crates/stellar-ledger/src/test_fixtures/speculos.rs

```diff
@@ -3,6 +3,8 @@ use testcontainers::{core::WaitFor, Image, ImageArgs};
 
```

Comment:
"test fixtures" are usually files that are there to support tests, so things like binaries, or generated code to regenerate to compare against. So I'd keep code supporting the tests in a tests/test directory, while keep artifacts that tests use in the test_fixtures directory.